src/plugins/proposal.ts

```diff
@@ -158,7 +158,7 @@ export interface IProposalState {
 
 export abstract class Proposal<TProposalState extends IProposalState> extends Entity<
   TProposalState
-> {
+  > {
```

Comment:
please remove the additional space here...

Reply:
fixed